configuration-api/main/mongo/alarms.js

```diff
@@ -1,12 +1,102 @@
 const fs = require('fs');
 const path = require('path');
+const mongoose = require('mongoose');
+const config = require('../config');
+const connection = mongoose.createConnection(config.getConfig().mongo_db);
```

Comment:
what happens if the connection goes down? will we only get a fresh working connection next time we bounce Node? If not, what's going to repair a broken connection?

Reply:
GraphQL is managing that, an error on the ui would be displayed and when mongo is back online the app is going to be reloaded automatically